models/stocks.js

```diff
@@ -38,7 +40,80 @@ const fetchStocks = async () => {
   }
 }
 
+/**
+ * Fetches the user stocks
+ * @return {Promise<userStocks|object>}
+ */
+const fetchUserStocks = async (userId, stockId = null) => {
+  try {
+    let userStocksRef = ''
+    const query = userStocksModel.where('userId', '==', userId)
```

Comment:
What about the case when the given userId is not found? Example, If I search for stocks of Elon Musk

I understand this function is for self user, but someone in future will use this function to get someone else's stock and they may not consider this case when the user provided is wrong.

Reply:
I understand what you mean here and that makes sense, but I think when in the future we use this, we can add this feature .. what say?

Reply:
As per our discussion, closing this .. we do need this check here.